packages/autocomplete-preset-algolia/src/search/fetchAlgoliaResults.ts

```diff
@@ -28,13 +28,30 @@ export interface SearchParams {
   userAgents?: UserAgent[];
 }
 
+function getSearchClientCredentials(searchClient: any): [string, string] {
+  if (searchClient.transporter) {
+    // searchClient v4
```

Comment:
Anything to do for v5?

Reply:
Not sure. it's coming from [this util](https://github.com/algolia/instantsearch/blob/master/packages/instantsearch.js/src/lib/utils/getAppIdAndApiKey.ts) in instantsearch.js. @Haroenv do you know if the `transporter` is different in v5?

Reply:
v5 uses transporter too